spec/classes/package_spec.rb

```diff
@@ -24,6 +24,7 @@
           'revision'                      => '01',
           'version'                       => '2.11.2',
           'download_folder'               => '/srv',
+          'md5sum'                        => '',
```

Comment:
You are almost right.
 
md5sum can not be used with undef value, that will cause during test:
```
ruby -c spec/classes/package_spec.rb
spec/classes/package_spec.rb:27: syntax error, unexpected keyword_undef
          'md5sum'                        => undef,
                                                  ^
spec/classes/package_spec.rb:115: syntax error, unexpected keyword_end, expecting '}'
rake aborted!
Command failed with status (1): [ruby -c spec/classes/package_spec.rb...]
 ```
caused by undef can not be used in hash
To be tested unset ```md5sum``` is enough  and check if the ```source_hash``` is nil when ```wget::fetch``` is called.

I add a commit with this.